client/src/pages/admin/authorizationgroup/Form.js

```diff
@@ -104,18 +104,24 @@ const AuthorizationGroupForm = ({ edit, title, initialValues }) => {
                   name="description"
                   component={FieldHelper.InputField}
                   componentClass="textarea"
-                  maxLength={Settings.maxTextFieldLength}
+                  maxLength={
+                    Settings.maxTextFieldLength?.authorizationGroup
+                      ?.description || Settings.maxTextFieldLength.default
```

Comment:
Too DRY (same expression is repeated three times); extract into a const, and perhaps provide a util method `getMaxTextFieldLength(setting)` that when called as
`getMaxTextFieldLength(Settings.maxTextFieldLength?.authorizationGroup?.description)` will either return the setting from the dictionary if defined, or the default.